project/project-info.conf

```diff
@@ -14,6 +14,7 @@ project-info {
       text: "Github issues"
     }
     release-notes: {
+      # TODO: Correct url needs to be provided
```

Comment:
Added another `TODO`. Release notes page needs to be added in future. Shall I create a task / issue?

Reply:
remove the TODO, this problem URL is easily found when we come back to remove the final akka refs